uc-cdis/gen3-code-vigil
# Migration - indexing page tests

Link to JIRA ticket if there is one: [GQE-227](https://ctds-planx.atlassian.net/browse/GQE-227) - Migrating Indexing Page UI tests into new framework.
Test scenarios - 
1. Upload and download valid manifest via Indexing page UI
2. Upload invalid manifest and expect indexing sower job to fail

### New Features

### Breaking Changes

### Bug Fixes

### Improvements

### Dependency updates

### Deployment changes



[GQE-227]: https://ctds-planx.atlassian.net/browse/GQE-227?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
